src/internalBinding/timers.cc

```diff
@@ -57,4 +56,4 @@ JSFunctionSpec InternalBinding::timers[] = {
   JS_FN("enqueueWithDelay", enqueueWithDelay, /* nargs */ 2, 0),
   JS_FN("cancelByTimeoutId", cancelByTimeoutId, 1, 0),
   JS_FS_END
-};
```

Comment:
Let's perhaps not mark this file as part of this commit solely for an extra line